Documentation/Localization.ru_RU/About.rst

```diff
@@ -0,0 +1,39 @@
+.. include:: /Includes.rst.txt
+
+
+.. _introduction:
+
+.. _about:
+
+================
+Об этом руководстве
+================
```

Comment:
These lines must always be as long or longer then the text

Reply:
Done in the latest update